File: WebApp/Models/AB/AB0038_VM.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class AB0038VM : JCLib.Mvc.BaseModel
    {
        public string SEQ { get; set; } //順序
        public string CNAME { get; set; } //中文名稱
        public string ENAME { get; set; } //英文名稱
        public string CHK { get; set; } //選擇FLAG
        public string IP { get; set; } //使用者IP
    }
}

File: WebApp/Scripts/app/model/AB0038VM.js
Ext.define('WEBAPP.model.AB0038VM', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'SEQ', type: 'string' }, //順序
        { name: 'CNAME', type: 'string' }, //中文名稱
        { name: 'ENAME', type: 'string' }, //英文名稱
        { name: 'CHK', type: 'date' }, //選擇FLAG
        { name: 'IP', type: 'date' }, //使用者IP
    ]
});